Eldamarth/reactTest
# Header and form positioning

#### What’s this PR do?

Adds header to App, and basic styling/positioning to Form and catlist display. Additionally removed hard-coded cat data and added function to delete unwanted cat.

#### Where should the reviewer start?

Changes have been made in App and Form js and css.

#### How should this be manually tested?

Filling in the form and hitting the button in the form should add a catbox with the appropriate information, clicking the delete cat button should remove it from the display and from the state of app. 

#### Any background context you want to provide?

Normally the display of the catboxes would be in a separate component from App, so the delete/remove function would have to be passed down as a prop like we did with the addCat function and the Form. 

#### What are the relevant tickets?
See below.
#### Screenshots (if appropriate)
#### Questions:
# Implements/Fixes:
* What issue does this close?
#5
